lib/MusicBrainz/Server/Data/URL.pm

```diff
@@ -34,6 +34,7 @@ my %URL_SPECIALIZATIONS = (
     'CDBaby'              => qr{^https?://(?:www.)?cdbaby.com/}i,
     'ChangeTip'           => qr{^https?://(?:www.)?changetip.com/tipme/}i,
     'CiNii'               => qr{^https?://(?:www.)?ci.nii.ac.jp/}i,
+    'CPDL'                => qr{^https?://(?:www.)?cdpl.org/wiki/}i,
     'Commons'             => qr{^https?://commons.wikimedia.org/wiki/File:}i,
```

Comment:
but I don't think a score link will ever need to match anything out of  http://www2.cpdl.org/wiki/ , would it ?